bundles/markup/index.rst

```diff
@@ -0,0 +1,72 @@
+MarkupBundle
+============
+
+The MarkupBundle provides the feature of declaring different "relations" as
+html-tag. This tags will be automatically parsed before the response will be
+send.
+
+Example
+-------
+
+This example contains a sulu-related example. The tag ``sulu:link`` represents
+a link to another page. This tag will be replaces via a valid anchor where the
+href links to the page with given UUID.
```

Comment:
`href attribute contains the UUID of the page`, and put `href` in double backticks.